components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -205,7 +207,10 @@ func (a *awsDeployment) validateConfigFields() *list.List {
 func (a *awsDeployment) validateEnvFields() *list.List {
 	errorList := list.New()
 	if len(a.config.Aws.Config.Profile) < 1 {
```

Comment:
in the config, specifically let user know through comment or through docs that this "Profile" field decides whether local credentials will be used or IAM role will be used

Reply:
Added changes in documentation